scripts/convert_and_dispatch_genotypes.py

```diff
@@ -13,7 +13,6 @@
 default_fai = join(etc_path, 'genotype_32_SNPs_genome_600bp.fa.fai')
 default_flank_length = 600
 
-logger = log_cfg.get_logger(__name__)
```

Comment:
We can remove the import of `logging_default` as well, since we're removing this logger